CODEOWNERS

```
@@ -12,6 +12,7 @@
 /incubator/opa/ @sudermanjr
 /incubator/ro-cert-manager/ @sudermanjr
 /incubator/route53sync/ @sudermanjr
+/incubator/oom-event-generator/ @sudermanjr
```

you put yourself as the maintainer in the Chart.yaml, so I was thinking more along the lines of:

```suggestion
/incubator/oom-event-generator/ @denraf 
```

---

Done